transcom/mymove
# B-19975-KB-INT

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19975)
## [Original INT PR](https://github.com/transcom/mymove/pull/12756)

## Summary

Small UI change requested

### How to test

1. As a customer with an approved PPM, create trip, set, and expense documents. Take a screen shot or notate the values. Submit.
2. As a closeout counselor, view the move and review the documents. Alter different amounts of stuff and approve/exclude (do not reject, as this will send the shipment back to the PPM Document submission stage) and confirm review is complete when you reach the end.
3. As a customer, view your move and confirm you can now see a "View Closeout Feedback" button. Click that and find yourself at the Feedback review page. Compare the values/adjusted values to the values you input. Check that it works as AC requests, that it accurately reflects the white paper, and try and break it.
4. Confirm that the total amounts next to the header are no longer displayed in parens and instead displayed like this
![Screenshot 2024-06-14 at 10 07 39 AM](https://github.com/transcom/mymove/assets/147535220/d80347dc-d28d-4193-9cea-e1b54c334e71)
